clean.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: clean [options]"
+    echo "Cleans the local dev environment."
+    echo
+    echo "  -b     Delete bin directory"
+    echo "  -p     Delete packages directory and NuGet package caches"
+    echo "  -a     Remove all untracked files in the working tree and delete NuGet package caches"
+    exit 1
+}
+
+# Parse arguments
+
+if [ $# == 0 ]
+then
+    echo "Please specify at least one option."
+    echo "Use 'clean -h' for help."
+    exit 1
+fi
+
+while [[ $# > 0 ]]
+do
+    opt="$1"
+    case $opt in
+        -h|--help)
+        usage
+        ;;
+        -b)
+        CleanBinDir=true
+        ;;
+        -p)
+        CleanPackagesDir=true
+        CleanPackagesCache=true
```

Comment:
I would only use the CleanPackagesDir variable. Both are related so every time you want to delete the packages folder you have to delete the cache.
And if you are cleaning the working tree, you could also set the CleanPackagesDir =true